static/app/views/projectInstall/platformIntegrationSetup.tsx

```diff
@@ -38,7 +38,7 @@ class PlatformIntegrationSetup extends AsyncComponent<Props, State> {
   getDefaultState() {
     return {
       ...super.getDefaultState(),
-      installed: false,
```

Comment:
@Neo-Zhixing why is the default for installed now `true`?

Reply:
Oops, good catch. Thanks!